tests/viet_kwik_e_mart/entities/store_name/mapping.json

```diff
@@ -0,0 +1,130 @@
+{
+  "entities": [
+    {
+      "id": "1",
+      "cname": "23 Elm Street",
```

Comment:
These entities will not make sense in vietnamese, so they shouldnt be added here. Either convert them to Vietnamese or maybe remove this entity?

Reply:
Done!